docs/NEW_LANGUAGE_FEATURE_INTEGRATION_PROCESS.md

```diff
@@ -127,18 +143,30 @@ These checks serve as references for designing and implementing new checks.
   new tokens involving identifiers, explore existing checks related to naming conventions.
   We can create a new similar check to enforce naming conventions for the new feature.
 
-## How to Create Tracker Issue
+- **Sizes**: If we have a check that enforces a limit on the number of parameters
+  a method can have, we created a similar check for the number of record components
+  to ensure that records do not have an excessive number of components.
+
+---
+
+## 2. How to Create Tracker Issue
 
 The tracker issue should document and share the results of the analysis
 conducted according to the outlined procedures in this document.
 It should include sections for each step of the analysis with the result
 of all of our actions. These tracker issues should be created for each
 new language feature, demonstrating our due diligence
 in integrating the feature into Checkstyle.
+See [#14961](https://github.com/checkstyle/checkstyle/issues/14961) and
+[#14942](https://github.com/checkstyle/checkstyle/issues/14942) for reference.
+
+---
 
-## How to Open Child Issues
+## 3. How to Open Child Issues
 
 Using your findings from the tracker issue, open child issues for each check that needs
```

Comment:
Same.

`Using your findings from the tracker issue` => `Using the findings from the tracker issue`

Reply:
done